httpd/handler.go

```diff
@@ -85,6 +85,10 @@ func NewHandler(s *influxdb.Server, requireAuthentication bool, version string)
 			"ping",
 			"GET", "/ping", h.servePing,
 		},
+		route{ // Tell data node to run CQs that should be run
+			"process_continuous_queries",
+			"POST", "/process_continuous_queries", h.serveProcessContinuousQueries,
```

Comment:
You may need to add the gzip (false?) flag here, due to some recent changes by @corylanou .